utils/lists.css

```diff
@@ -0,0 +1,28 @@
+.list {
+    display: flex;
+    flex-direction: column;
+}
+
+.basic-list {
+    border: 1px solid black;
+    padding: 16px;
+}
+
+.basic-list:hover {
+    cursor: pointer;
+    background-color: var(--primary);
+    color: #ffffff;
```

Comment:
It is good that you have used CSS variable for color in line 13, it will be good if you do it same across all the CSS files.